packages/eas-cli/src/commands/build/create.ts

```diff
@@ -35,9 +37,21 @@ export default class BuildCreate extends Command {
     }),
   };
 
-  async run() {
+  async run(): Promise<void> {
     const { flags } = this.parse(BuildCreate);
     await ensureLoggedInAsync();
+
+    const projectDir = (await findProjectRootAsync()) ?? process.cwd();
+    const projectId = await getProjectIdAsync(projectDir);
+
+    if (!(await isEasEnabledForProjectAsync(projectId))) {
+      log.error(
+        'Your account does not have access to Expo Application Services (EAS) features. Please enroll in EAS to give it a try.'
```

Comment:
Maybe a link here where you can "enroll in EAS"?

Reply:
@brentvatne what should be the link?

Reply:
@fson I'll open an issue to add the link after merging this PR.